osf/models/node.py

```diff
@@ -457,37 +433,25 @@ def collecting_metadata_qs(self):
     def collecting_metadata_list(self):
         return list(self.collecting_metadata_qs)
 
-    @property
-    def is_preprint(self):
-        # TODO: This is a temporary implementation.
-        if not self.preprint_file_id or not self.is_public:
-            return False
-        if self.preprint_file.target == self:
-            return self.has_submitted_preprint
-        else:
-            self._is_preprint_orphan = True
-            return False
-
     @property
     def has_submitted_preprint(self):
-        return self.preprints.exclude(machine_state=DefaultStates.INITIAL.value).exists()
-
-    @property
-    def is_preprint_orphan(self):
-        """For v1 compat"""
-        if (not self.is_preprint) and self._is_preprint_orphan:
-            return True
-        if self.preprint_file:
-            return self.preprint_file.is_deleted
-        return False
+        # Attached to a submitted preprint
+        return self.preprints.filter(
+            deleted__isnull=True).exclude(machine_state=DefaultStates.INITIAL.value).exists()
```

Comment:
Used for legacy front-end, `has_submitted_preprint` and `has_published_preprint`, meaning this node is a supplemental node for a submitted preprint or published prepirnt.